Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -42,10 +42,19 @@ protected class MemoryCacheItem
             public BitmapImage Image { get; set; }
         }
 
+        private class ConcurrentTask
+        {
+            public Task<BitmapImage> Task { get; set; }
+
+            public bool IsAssureOnly { get; set; }
```

Comment:
I agree. What about IsAssureCachedOnly? Because download does not happen if image is already in the cache.